rust-src/concordium_base/src/smart_contracts.rs

```diff
@@ -212,3 +212,13 @@ impl Deserial for Parameter {
         Ok(Parameter { bytes })
     }
 }
+
+/// Conversion between essentially duplicated types.
+impl From<OwnedParameter> for Parameter {
```

Comment:
It would be nice to unify these types.
Both are probably used a lot by now, so we should probably discuss this